tsconfig.json

```diff
@@ -7,6 +7,8 @@
     "declaration": true,
     "strict": true
   },
-  "include": ["src/**/*.ts"],
-  "exclude": ["node_modules"]
+  "include": ["src/**/*.ts", "src/*.ts"],
```

Comment:
`src/**/*.ts` is enough, `src/*.ts` is redundant. 
Why do you need that change?